libs/types-common/src/Data/Misc.hs

```diff
@@ -220,7 +220,7 @@ instance Cql (Fingerprint a) where
 
 newtype PlainTextPassword = PlainTextPassword
     { fromPlainTextPassword :: Text }
-    deriving (ToJSON)
+    deriving (Eq, Show, ToJSON)
```

Comment:
I'd advise against deriving a `Show` instance here, just in case a `PlainTextPassword` ends up in logs

Reply:
:+1:

embarassing.  thanks for catching this!